local/config/finna/WayfinderService.ini.sample

```diff
@@ -2,20 +2,23 @@
 
 [General]
 enabled = false
-adapter = "WayfinderSampleAdapter"
 
 [sample]
 wf_url = "http://wf-hambib.stg.easyscreen.io/includes"
 marker_url = "http://wf-hambib.stg.easyscreen.io"
+adapter = "WayfinderSampleAdapter"
 
 [piki]
 wf_url = "http://wf-piki.stg.easyscreen.io/includes"
 marker_url = "http://wf-piki.stg.easyscreen.io"
+adapter = "WayfinderPikiAdapter"
 
 [vari]
 wf_url = "http://wf-vari.stg.easyscreen.io/includes"
 marker_url = "http://wf-vari.stg.easyscreen.io"
+adapter = "WayfinderVariAdapter"
```

Comment:
I don't think it makes sense for the sample config to point to non-existent adapters.